tests/acceptance/tags/PageTagCest.php

```diff
@@ -220,14 +234,21 @@ public function testBulkEditUsingDefinedTag(AcceptanceTester $I)
 
 		// Iterate through Pages to run frontend tests.
 		foreach ($pageIDs as $pageID) {
+			// Programmatically create a subscriber in ConvertKit.
+			// Must be a domain email doesn't bounce on, otherwise subscriber won't be confirmed even if the Form's
+			// "Auto-confirm new subscribers" setting is enabled.
+			// We need the subscriber to be confirmed so they can then be tagged.
+			$emailAddress = $I->generateEmailAddress('n7studios.com');
```

Comment:
Is there a CK-domain we can use instead?

Reply:
I found that `convertkit.com` domains would bounce (expected, as these are generated email addresses with unique strings).  If there's a particular CK domain you have that accepts wildcards and might not bounce, let me know and I can use that instead.